djangoevents/unifiedtranscoder.py

```diff
@@ -80,11 +81,19 @@ def deserialize(self, stored_event):
         event_attrs = self._json_decode(stored_event.event_data)
 
         # Reinstantiate and return the domain event object.
+        defaults = {
+            'entity_id': stored_event.aggregate_id,
+            'entity_version': stored_event.aggregate_version,
+            'domain_event_id': stored_event.event_id,
+        }
+
+        if adds_schema_version_to_event_data():
```

Comment:
Wait wait ```adds_schema_version_to_event_data``` returns override_settings, what is conditional here so that we need the if

Reply:
Please change the name of the conditional-function here

Reply:
It's a different function with the same name, I'll change it.